bundles/org.openhab.binding.miio/src/main/resources/database/nwt.derh.wdh318efw1.json

```diff
@@ -0,0 +1,193 @@
+{
+	"deviceMapping": {
+		"id": [
+			"nwt.derh.wdh318efw1"
+		],
+		"channels": [
+			{
+				"property": "on_off",
+				"friendlyName": "Power",
+				"channel": "power",
+				"type": "Switch",
+				"refresh": true,
+				"ChannelGroup": "Power",
+				"actions": [
+					{
+						"command": "set_power",
+						"parameterType": "ONOFF"
+					}
+				],
+				"category": "switch",
+				"tags": [
+					"Switch"
+				]
+			},
+			{
+				"property": "mode",
+				"friendlyName": "Mode",
+				"channel": "mode",
+				"type": "String",
+				"refresh": true,
+				"ChannelGroup": "actions",
+				"actions": [
+					{
+						"command": "set_mode",
+						"parameterType": "STRING"
+					}
+				],
+				"tags": [
+					"Control"
+				]
+			},
+			{
+				"property": "auto",
+				"friendlyName": "Auto humidity",
+				"channel": "autohumidity",
+				"type": "Number:Dimensionless",
+				"unit": "PERCENT",
+				"refresh": true,
+				"ChannelGroup": "actions",
+				"actions": [
+					{
+						"command": "set_auto",
+						"parameterType": "NUMBER"
+					}
+				]
+			},
+			{
+				"property": "fan_st",
+				"friendlyName": "Fan Speed",
+				"channel": "fanspeed",
```

Comment:
Huh, why do fan speed and fan status have opposite channel and property names?

Reply:
that must be an error. I fixed it